lib/puppetfile-updater/task.rb

```diff
@@ -70,7 +70,7 @@ def define(args, &task_block)
           end
 
           # Update from GitHub
-          aug.match("/files/Puppetfile/*[git=~regexp('.*/#{@user}/.*')]").each do |mpath|
```

Comment:
LGTM, but maybe `[/-]` is a bit simpler.

Reply:
@raphink updated it